docs/src/main/sphinx/sql/set-path.rst

```diff
@@ -0,0 +1,54 @@
+========
+SET PATH
+========
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET PATH path-element[, ...]
+
+Description
+-----------
+
+Define a collection of paths to functions or table functions in specific
+catalogs and schemas for the current session.
+
+Each path-element uses a period-separated syntax of to specify the catalog name
+and schema location ``<catalog>.<schema>`` of the function or only the schema
+location ``<schema>`` in the current catalog. The current catalog is set with
+:doc:`use` or as part of a client tool connection. Catalog and schema must
+exist.
+
+The defined path is used in addition to the global, built-in path.
+
+Examples
+--------
+
+The following example sets a path to access functions in the ``system`` schema
+of the ``example`` catalog::
+
+    SET PATH example.system;
+
+The catalog uses the PostgreSQL connector, and you can therefore use the
+:ref:`query table function <postgresql-query-function>` directly, without the
+full catalog and schema access path::
```

Comment:
"access path" in this sentence is confusing. The path is a set of catalog and schemas that are inspected to resolve a function. Here you're using "access path" to mean "coordinates" of the function. I would just say "without the full catalog and schema qualifiers".